set.mm

```diff
@@ -62625,16 +62625,41 @@ empty set when it is not meaningful (as shown by ~ ndmfv and ~ fvprc ).
   $}
 
   ${
-    $d x A $.  $d x B $.  $d x F $.  $d x G $.
-    $( Equality theorem for function value.  (Contributed by NM,
-       29-Dec-1996.) $)
+    $d x y z w A $.  $d x y z w B $.  $d x y z w F $.  $d x y z w G $.
+    $( Equality theorem for function value.  (Contributed by NM, 29-Dec-1996.)
+       Avoid ~ ax-12 .  (Revised by Gino Giotto, 28-May-2024.) $)
     fveq1 $p |- ( F = G -> ( F ` A ) = ( G ` A ) ) $=
+      ( vx vy vz vw wceq cv wbr cio cfv cab cuni wcel wex abbidv df-uni 3eqtr4g
```

Comment:
The current version of fveq1 is already free of ax-12, and it is shorter.  Here is the list of axioms:
ax-mp  ax-1  ax-2  ax-3  ax-gen  ax-4  ax-5  ax-6  ax-7  ax-8  ax-9  ax-ext

Reply:
Ah true, this was already automatically removed in the latest PR https://github.com/metamath/set.mm/pull/4020 and my version was outdated.